packages/flutter/lib/src/material/checkbox.dart

```diff
@@ -66,7 +74,13 @@ class Checkbox extends StatefulWidget {
   /// change state until the parent widget rebuilds the checkbox with the new
   /// value.
   ///
-  /// If null, the checkbox will be displayed as disabled.
+  /// If this callback is null, the checkbox will be displayed as disabled
+  /// and will not respond to input gestures.
+  ///
+  /// When the checkbox is tapped, if [triState] is false (the default) then
+  /// the onChanged callback will be applied to `!value`. If [triState] is
```

Comment:
not sure what you mean by "applied to" in this paragraph

Reply:
https://en.wikipedia.org/wiki/Apply

Reply:
Ah, ok. I think that phrasing would be confusing to our less mathematically-inclined users. I'd probably just say "will be called with the value" or some such.